### ISSUE TITLE
Is it possible to train on stronger depth estimator?

### ISSUE THREAD

**Issue by LaFeuilleMorte** [DepthAnything/PromptDA#10]
Hi, That's a great idea to get accurate metric depth based on lidar sparse depth. While the depth-anything models are great in some cases, a recent work : https://github.com/microsoft/moge might perform better on the depth map accuracy than depth anything. Is it possible to train your model on Moge to obtain better reconstruction results?

**Comment by haotongl** [comment]
Yes, it's possible in theory and in practice. We currently have no such plans.
